mw-webapp/src/component/text/Text.tsx

```diff
@@ -0,0 +1,60 @@
+import clsx from "clsx";
+import styles from "src/component/text/Text.module.scss";
+
+/**
+ * Type of  text variant styles
+ */
+
+export type ThemeVariant = "clear" | "outline";
+
+/**
+ * Type of size text styles
+ */
+
+export type TextSize = "m" | "l" | "xl";
+
+/**
+ * Params for {@link Text}
+ */
+interface TextProps {
+
+  /**
+   * Displayed text
+   */
+  text: string | number;
+
+  /**
+   * Displayed size text
+   */
+  size?: TextSize;
+
+  /**
+   * Displayed variant theme text
+   */
+  variant?: ThemeVariant;
+
+  /**
+   * Data attributes for cypress testing
+   */
+  cy?: string;
+}
+
+/**
+ * Component Text
+ */
+export const Text = (props: TextProps) => {
+  return (
+    <div data-cy={props.cy}>
+      {props.text &&
```

Comment:
props.text is not optional so it's overall. Am I wrong?

Reply:
obligatory text.props